collect_app/src/test/java/org/odk/collect/android/mainmenu/RequestPermissionsViewModelTest.kt

```diff
@@ -0,0 +1,67 @@
+package org.odk.collect.android.mainmenu
+
+import android.Manifest.permission.POST_NOTIFICATIONS
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.any
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import org.odk.collect.permissions.PermissionsChecker
+import org.odk.collect.settings.InMemSettingsProvider
+import org.odk.collect.settings.keys.MetaKeys
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+@Config(sdk = [33])
```

Comment:
This is redundant since by default robolectric runs our code against the targetSdkVersion specified in the manifest. So it will never be lower than 33.